python/python-copr.spec

```diff
@@ -147,6 +147,7 @@ BuildRequires: python3-pytest
 BuildRequires: python3-requests-gssapi
 BuildRequires: python3-future
 BuildRequires: python3-filelock
+BuildRequires: pyproject-rpm-macros
```

Comment:
This seems redundant.  Per guidelines, we need python3-devel:
https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/

Reply:
Hmm, I am not sure why we actually need all the other requirements that should be automatically generated.

Reply:
ACK.  I think we can keep it (seems like a distro glitch).